src/pages/api/lanes/[...slug].ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import { promises as fs } from "fs";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  request: NextApiRequest,
+  response: NextApiResponse
+) {
+  const slug = request.query.slug;
```

Comment:
It should be okay to make the name of this file `src/pages/api/lanes/[slug].ts` and on this line you can extract the value directly like `const { slug } = request.query;`. The `[...slug]` works as a catch-all but do we need that?

Reply:
We do not need it since we don't anticipate data being nested any further in the `data/lanes` folder (as discussed)